Lib/pathlib.py

```diff
@@ -851,8 +851,8 @@ def samefile(self, other_path):
         return os.path.samestat(st, other_st)
 
     def iterdir(self):
-        """Iterate over the files in this directory.  Does not yield any
-        result for the special paths '.' and '..'.
+        """Iterate over the files in this directory.  Does not include
+        the special paths '.' and '..'.
         """
```

Comment:
iterdir also iterates over subdirectories, not just files.

Maybe we could steal from the wording in the docs:
```suggestion
        """Yields path objects of the directory contents.
        The children are yielded in arbitrary order, and the
        special entries '.' and '..' are not included.
        """
```